xbmc/guilib/IWindowManagerCallback.h

```diff
@@ -26,4 +26,5 @@ class IWindowManagerCallback
   virtual void FrameMove(bool processEvents, bool processGUI = true) = 0;
   virtual void Render() = 0;
   virtual void Process() = 0;
+  virtual bool IsRenderGuiEnabled() = 0;
```

Comment:
rename this to `GetRenderGUI`, the CApplication has already this method

Reply:
Humm... having problems  making  `GetRenderGUI`  a member of `IWindowManagerCallback`